app/src/test/kotlin/com/intive/picover/auth/repository/AuthRepositoryTest.kt

```diff
@@ -151,5 +153,9 @@ class AuthRepositoryTest : ShouldSpec(
 			result shouldBeEqual Profile(userPhoto, "Marian Kowalski", userEmail)
 			slot.captured.displayName!! shouldBeEqual "Marian Kowalski"
 		}
+
+		should("not set storageReference WHEN class is created") {
```

Comment:
part of the name "not set" seems to be incorrect, we're not checking any "set" here, it's rather "call"

Reply:
@MarekMacko Updated

Reply:
Nice, thank you ;)